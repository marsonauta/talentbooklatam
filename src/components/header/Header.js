import React from 'react'
import './Header.css'

const Header = (props) => {
    return <header>
        <h1><a href="/"><i className="fas fa-plane-departure"></i></a></h1>
        <nav>
            <a href="en.html">ğŸ‡ºğŸ‡¸<span className="mobile-no"> English</span></a>
            <a href="pt.html">ğŸ‡§ğŸ‡·<span className="mobile-no"> PortuguÃªs</span></a>
        </nav>
    </header>
}

export default Header