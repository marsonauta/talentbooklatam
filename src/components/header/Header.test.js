import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header'

describe("Header component", () => {
  test("Matches the snapshot", () => {
    const header = renderer.create(<Header />).toJSON()
    expect(header).toMatchSnapshot();
  });
});