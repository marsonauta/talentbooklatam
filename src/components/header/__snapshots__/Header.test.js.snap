// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Matches the snapshot 1`] = `
<header>
  <h1>
    <a
      href="/"
    >
      <i
        className="fas fa-plane-departure"
      />
    </a>
  </h1>
  <nav>
    <a
      href="en.html"
    >
      ğŸ‡ºğŸ‡¸
      <span
        className="mobile-no"
      >
         English
      </span>
    </a>
    <a
      href="pt.html"
    >
      ğŸ‡§ğŸ‡·
      <span
        className="mobile-no"
      >
         PortuguÃªs
      </span>
    </a>
  </nav>
</header>
`;
